recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,10 @@
+do_install_append() {
```

Comment:
This append process should be done only when MACHINE is rpi3, shouldn't it?

Reply:
This process need to rpi3 b+.  However, this file is only installed in FILES_${PN}-bcm43455 pacakge.

Reply:
I think this process is executed also when MACHINE is qemuarm or other machine, and it results in build error. To avoid it, we need do_install_append_raspberrypi3 or do_install_append_raspberrypi3-64.

Reply:
@hiraku-wfs Okay, I renamed it.